### ISSUE TITLE
Support for Windows 11?

### ISSUE THREAD

**Issue by hyperfear11** [artman41/guitarsniffer#18]
Think anyone can update this to support Windows 11? Doesn't seem to open at all. Works fine on Windows 10. Simple fix hopefully?

**Comment by Dunkalunk** [comment]
Sorry it took so long to respond.  I got a new laptop and still hadn't set up Clone Hero or the sniffer.

I was able to install all the required software and the sniffer opens and works great on Windows 11.  I have noticed in the past that after Windows updates, sometimes I have to reinstall the WinPcap software if the sniffer closes immediately.  Not sure why that is, but that usually does the trick for me!

Hope this helps!

**Comment by OWPanda** [comment]
> Sorry it took so long to respond. I got a new laptop and still hadn't set up Clone Hero or the sniffer.
> 
> I was able to install all the required software and the sniffer opens and works great on Windows 11. I have noticed in the past that after Windows updates, sometimes I have to reinstall the WinPcap software if the sniffer closes immediately. Not sure why that is, but that usually does the trick for me!
> 
> Hope this helps!

WinPcap works on windows 10??

**Comment by Dunkalunk** [comment]
Yes I have installed all the required programs from the instructions on both Windows 10 and 11
